supervision/README.md

```diff
@@ -42,6 +44,22 @@ During different periods, some activities will be more pronounced than others (e
        - There may also be TODOs for your supervisor. Prefix them with their initials, e.g., "[KB]".
   * If there are items that need preparation from the supervisor's end before the meeting, i.e., the questions/blockers block is not empty, make sure you share the agenda at least 24 hours before the meeting (by emailing the link to your Google document).
 
+### Supervisor roles and responsibilities 
+* Each thesis group has a course coordinator (fagligansvarlig) and a supervisor. 
+* The supervisor can be internal if the thesis work is carried out at the UiS or external if the thesis work is carried out at an external company.
+* Course coordinator and internal supervisor can be different people in case PhD students are the internal supervisors.
```

Comment:
"The course coordinator and internal supervisor can be the same person (e.g., professor) or can be two different people (e.g., internal supervisor is a PhD student)."

Reply:
done